packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -209,7 +220,7 @@ export class FabricTestLedgerV1 implements ITestLedger {
       const hostPort = await Containers.getPublicPort(privatePort, cInfo);
       ccp.peers["peer0.org1.example.com"].url = `grpcs://localhost:${hostPort}`;
     }
-    {
+    if (JSON.stringify(ccp).indexOf("peer1.org1.example.com") > -1) {
```

Comment:
Prefer to have this check implemented in a more deterministic way. This can fail if a new, unrelated property gets added to the ccp object (unlikely, but can happen and therefore we don't want that edge case to be possible at all).

Example:

```typescript
const ccp = {
    description: "New ccp edition that omits peer1.org1.example.com",
    peers: {
        "peer0.org1.example.com": { ... }
    }
};

// now if you check here for the presence of `peer1.org1.example.com` 
// it will return true, but there is no such peer and it will start crashing
```

Reply:
Incorporated the changes. Please have a re-look.